apps/chat_websocket/lib/chat_websocket/channels/rooms_channel.ex

```diff
@@ -20,11 +23,26 @@ defmodule ChatWebsocket.RoomsChannel do
   def handle_info(:after_join, socket) do
     %{user: user} = socket.assigns
 
+    push socket, "starred_messages_state", GetStarredMessages.call(user)
```

Comment:
One pattern that changed since you wrote this is only using `_state` for Presence data. In other words, this should become:

```elixir
push socket, "starred_messages", GetStarredMessages.call(user)
```